src/httprpc.cpp

```diff
@@ -236,6 +251,9 @@ bool StartHTTPRPC()
 #ifdef ENABLE_WALLET
     // ifdef can be removed once we switch to better endpoint support and API versioning
     RegisterHTTPHandler("/wallet/", false, HTTPReq_JSONRPC);
+
+    void JSONRPCRequestWalletResolver(JSONRPCRequest& jreq, const HTTPRequest& httpreq);
```

Comment:
use `extern`outside function scope?

Reply:
Isn't `extern` not needed (nor desirable in our codebase) for function references?